metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowExecutorManager.java

```diff
@@ -59,27 +59,25 @@ public WorkflowExecutorManager(
    */
   public void initiateConsumer() throws IOException {
     //For correct priority. Keep in mind this pre-fetches a message before going into handleDelivery
-    getRabbitmqChannel().basicQos(1);
+    getRabbitmqConsumerChannel().basicQos(1);
     //Auto acknowledge false(second parameter) because of Qos.
-    getRabbitmqChannel().basicConsume(rabbitmqQueueName, false, queueConsumer);
+    getRabbitmqConsumerChannel().basicConsume(rabbitmqQueueName, false, queueConsumer);
   }
 
   /**
-   * Adds a WorkfloExecution identifier in the distributed queue.
+   * Adds a WorkflowExecution identifier in the distributed queue.
    *
    * @param userWorkflowExecutionObjectId the WorkflowExecution identifier
    * @param priority the priority of the WorkflowExecution in the queue
    */
-  public void addWorkflowExecutionToQueue(String userWorkflowExecutionObjectId,
-      int priority) {
+  public void addWorkflowExecutionToQueue(String userWorkflowExecutionObjectId, int priority) {
     synchronized (this) {
```

Comment:
Synchronize on the channel instead of on this, just to be sure? In case we ever decide to use the channel elsewhere or we allow the existence of two instances of WorkflowExecutionManager?